openwisp_firmware_upgrader/admin.py

```diff
@@ -271,6 +277,22 @@ def readonly_upgrade_options(self, obj):
         )
 
 
+@admin.register(UpgradeOperation)
+class UpgradeOperationAdmin(ReadOnlyAdmin, BaseAdmin):
+    list_display = ["device", "status", "image", "modified"]
+    list_filter = ["status"]
+    search_fields = ["device__name"]
+    readonly_fields = ["device", "image", "status", "log", "modified"]
+    ordering = ["-modified"]
+    fields = ["device", "image", "status", "log", "modified"]
+
+    def has_add_permission(self, request):
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        return False
```

Comment:
Is this just for debugging or what? I don't see any progress bar in this admin page. Why is it being added?